python/paddleserver/paddleserver/test_model.py

```
@@ -49,24 +49,30 @@
         img = cv2.imread(os.path.join(model_dir, filename))
-        input_data = face_detect_preprocess(img)
-        request = {"instances": input_data}
-        response = server.predict(request)
-        faces = response["predictions"]
-        assert sum(face[1] > 0.5 for face in faces) == expected
 
-        # test v2 infer call
+        try:
+            input_data = image_preprocess(img)
+        except Exception as e:
+            assert False, f"Image processing failed: {str(e)}"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)